apps/fluent-tester/src/FluentTester/TestComponents/FocusZone/styles.ts

```diff
@@ -31,6 +32,7 @@ export const GridButton = Button.compose({
     root: View,
     icon: Icon,
     content: Text,
+    shadow: Shadow,
```

Comment:
Why are we adding this to FocusZone?

Reply:
I think something was complaining until I added this. I can double check if I need to keep it in

Reply:
Ah, it was because it was a Button slot. Probably don't need it anymore.